backend/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
remove me

Reply:
If we don't ignore this it will commit thousands of directories related to libraries that'll already be installed locally when you run `npm install`